### DISCUSSION TITLE
How to install with Symfony

### DISCUSSION THREAD

**Discussion by jzeilmaker** [ColorlibHQ/AdminLTE#4651] [General]
How do i install AdmineLTE with Symfony?
I did a composer require but not sure what to do next

**Comment by jzeilmaker** [comment]
run pm install admin-lte --save ?

**Comment by jzeilmaker** [comment]
Any one that can help me out? i just dont get it..

**Reply by REJack** [reply]
Sadly I don't use Symfony, may someone at stackoverflow can help you there 😞.

**Comment by eerison** [comment]
Maybe [Sonata](https://github.com/sonata-project) help you? this bundle uses this admin layout ;)
